yarn.lock

```diff
@@ -775,6 +775,15 @@
     fs-extra "^10.0.0"
     which "^2.0.2"
 
+"@electron-forge/maker-base@^6.0.4":
+  version "6.2.1"
+  resolved "https://registry.yarnpkg.com/@electron-forge/maker-base/-/maker-base-6.2.1.tgz#04dcc6163a2938480ed5c887fa2894646c7eebb8"
+  integrity sha512-LnvGtTJ/RNojKdUKktYEcbLqPggXdMBs1uscQRgXkI3XnVGdEi+/j5+Eg5ka4d6FnsaUkz//U5yhPtNFhDbNSw==
+  dependencies:
+    "@electron-forge/shared-types" "6.2.1"
```

Comment:
isn't it a bit weird that `maker-base` specifically wants the latest version of `shared-types` rather than `^6.0.4`?

Reply:
The `@electron-forge/*` packages are specifically version locked to each other, so that's why `@electron-forge/maker-base@6.2.1` only wants `@electron-forge/shared-types@6.2.1`.

Had some discussion going on about the best way to prevent the usage of `@electron-forge/maker-base@6.2.1` here (which might be harmless, but I'd prefer to avoid any subtle bugs, already enough foot guns). Opening #1381 to address that and this PR can rebase once that lands.

Reply:
nice, thanks :)